docs/content/getting-started/perform-list-users.mdx

```diff
@@ -92,11 +93,25 @@ This section will illustrate how to perform a <ProductConcept section="what-is-a
 </TabItem>
 </Tabs>
 
-## Step By Step
+## Step by step
+
+Consider the following model which includes a `user` can have a `reader` relationship with a `document`:
```

Comment:
I'm not sure whether this needs `that` to make the sentence?

```suggestion
Consider the following model which includes a `user` that can have a `reader` relationship with a `document`:
```

Reply:
Yes,  I think it does.  Thanks!